tests/ignite/metrics/test_metrics_lambda.py

```diff
@@ -149,7 +149,7 @@ def test_metrics_lambda_update():
     recall = Recall(average=False)
 
     def Fbeta(r, p, beta):
-        return torch.mean((1 + beta ** 2) * p * r / (beta ** 2 * p + r)).item()
+        return (1 + beta ** 2) * p * r / (beta ** 2 * p + r)
```

Comment:
Why this change ?

Reply:
As we agreed, `average=False` results in precision and recall becoming an array with length equal to number of classes. Had I not changed  that line, it would have returned the average of Fbeta for class zero and Fbeta for class one. Now it returns Fbeta array and I pick Fbeta for class one in line 215

Reply:
> As we agreed, average=False results in precision and recall becoming an array with length equal to number of classes.

It was the same behaviour previously as well. That's why `torch.mean(...).item()` was applied to compute an average. I think we should keep tests without much changes to ensure BC.